cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsTest.java

```diff
@@ -220,7 +225,7 @@ void exm125_careGaps_twoMeasuresByUrlAndId() {
 
     @Test
     void exm125_careGaps_error_wrongSubjectParam() {
-        assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
Question:  Just curious why you chose to do this?

Reply:
this structure change allows me to validate that the exact error thrown was the one the test was designed to catch, since there are multiple illegal arguments